setup.py

```diff
@@ -73,6 +73,7 @@ def get_extras_require():
             'sphinx_rtd_theme',
         ],
         'example': [
+            'allennlp',
```

Comment:
`allennlp` does not support Python 3.5. (c.f., https://pypi.org/project/allennlp/). Please exclude it from the installation targets if the Python version is 3.5.
In addition, please exclude `examples/allennlp` from the CI targets if the job is `examples-python35`.